recipes/pyldavis/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyldavis" %}
+{% set version = "2.1.0" %}
+{% set sha256 = "0fc2fb5d5fecc6b1a1bd97b82bea5ebe1102a5d6cf4edf4f4279a2209544a644" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest
+    - pytest-runner
+  run:
+    - python
+    - numpy >=1.9.2
+    - scipy >=0.18.0
+    - pandas >=0.17.0
+    - joblib >=0.8.4
+    - jinja2 >=2.7.2
+    - numexpr
+    - future
+    - funcy
```

Comment:
I believe `funcy` is not available via conda-forge or the default continuum channels, so you'll need to add this as a separate recipe.